src/store/tests/AutomatedTester.java

```diff
@@ -84,6 +83,10 @@ public void testAddProduct() {
 	public void testAll() {
 		testEnrollMember();
 		testAddProduct();
+		for (int index = 1; index <= 20; index++) {
+			Request.instance().setOrderId("O-" + index);
+			GroceryStore.instance().processOrder(Request.instance());
+		}
```

Comment:
Trying to understand what this does?

Reply:
When we just addProduct, the products are not physically on shelves, but rather just ordered. When order arrives, it needs to be processed. Method processOrder adds the quantities shipped to our stock on hand, products become available for check out, the order is marked as not outstanding and products are marked as not ordered. It became clear to me, that every product needs to carry a field called isOrdered. Once a product gets ordered, it cannot be ordered again, until its shipment arrives. The for loop you’re looking at processes each product’s order.